lib/at_host/at_host.c

```diff
@@ -292,13 +284,7 @@ static int at_host_init(const struct device *arg)
 		return -EINVAL;
 	}
 
-	err = at_notif_init();
-	if (err) {
-		LOG_ERR("Failed to initialize AT notifications, err %d", err);
-		return err;
-	}
-
-	err = at_notif_register_handler(NULL, response_handler);
+	err = nrf_modem_at_notif_handler_set(response_handler);
```

Comment:
Is this ok in at_host? I thought at_monitor is only component setting this handler but is at_host somehow a special case compared to other libraries that use at_monitor?

Reply:
@MirkoCovizzi we should instead register a monitor, to make sure this library an co-exist with others that want to receive AT notifications